packages/blocks/src/_common/embed-block-helper/embed-block-element.ts

```diff
@@ -67,6 +67,13 @@ export class EmbedBlockElement<
         return false;
 
       const blockComponent = anchorComponent as this;
+      const element = captureEventTarget(state.raw.target);
+
+      const canDrag =
+        blockComponent.contains(element) ||
+        !!element?.closest('affine-drag-handle-widget');
```

Comment:
`AFFINE_DRAG_HANDLE_WIDGET` will be better

Reply:
@Flrande Done, thanks for the review